app/controllers/api/v1/demographics/districts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::Demographics::DistrictsController < Api::ApiController
+  respond_to :json
+
+  def show
```

Comment:
This is a fairly large method. You may want to consider finding a way to clean this up by breaking up the conditional. The nice thing about this process is that it allows you to name different parts of the conditional in ways that make it more readable. As with everything I'm saying, this is just a suggestion! Do what works best for you.